test.tf

```diff
@@ -0,0 +1,47 @@
+variable "bucket" {
+  default = "aviltest"
+}
+
+variable "region" {
+  default = "us-east-1"
+}
+
+provider "aws" {
+  region = var.region
+  access_key = "AKIAWVIUOJZXDIRPSJU2"
```

Comment:
@Jlopezjlx this is a security issue. Please, review this.

Reply:
will make the changes

Reply:
Done, changes are up